specification/billing/resource-manager/Microsoft.Billing/preview/2020-12-15-privatepreview/examples/AddEnrollmentAccountByDepartment.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "api-version": "2020-12-15-privatepreview",
+    "billingAccountName": "{billingAccountName}",
+    "enrollmentAccountName": "{enrollmentAccountName}",
+    "departmentName": "{departmentName}",
+    "parameters": {
+      "accountOwner": "AccountName0@mail.com",
+      "accountName": "AccountName0",
+      "authType": "MicrosoftAccount",
+      "costCenter": "C0"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/enrollmentAccounts/{enrollmentAccountName}",
```

Comment:
Shouldn't this include department in its id, and basically match what the PUT was done on?